Resource recommendation guide for Promscale traces (#990)
timescale/docs: promscale/installation/recomm-guide.md

# Resource recommendations

> This section describes the compute and disk requirements that are recommended for Promscale, based on the ingest rate and retention.

Reviewer: ```suggestion
This section describes the compute and disk requirements that are 
recommended for Promscale, based on the ingest rate and retention.
```

**Disk size per day**: disk consumption per day based on the ingest rate.

**Uncompressed buffer**: disk consumed by uncompressed data before compression.

**Total disk size**: size of disk required to store the data based on the ingest rate and retention. 

<highlight type="note">

> You can calculate the `total disk size` based on rentention and ingest rate using the following formula:
> 

Reviewer: ```suggestion
You can calculate the `total disk size` based on retention and ingest rates with this formula:
```

# Resource recommendations
This section describes the compute and disk requirements that are 
recommended for Promscale, based on the ingest rate and retention.

**Disk size per day**: disk consumption per day based on the ingest rate.

**Uncompressed buffer**: disk consumed by uncompressed data before compression.

**Total disk size**: size of disk required to store the data based on the ingest rate and retention. 

<highlight type="note">
You can calculate the `total disk size` based on retention and ingest rates with this formula:

Total disk size = (Disk size per day based on the ingest rate * Retention in days) + 

> Uncompressed buffer based on the ingest rate.

Reviewer 2: + WAL

Reviewer 2: Otherwise it's not really total disk size :) or maybe document that WAL is not included for whatever reason

|100k samples/sec|90 days|~20 GB|~210 GB|~2 TB|-|
|200k samples/sec|90 days|~40 GB|~420 GB|~4 TB|-|

## Traces
Resource recommendation for ingestion through OTLP (OpenTelemetry Line Procotol) gRPC endpoint. 

### OpenTelemetry Line Protocol

> We recommend using the OpenTelemetry collector for ingesting the spans to Promscale by configureing the OTLP exporter and batch processor with below provided configuration this helps in retries for failed writes and in batching the write requests to Promscale:

Reviewer: ```suggestion
We recommend using the OpenTelemetry collector for ingesting the spans to 
Promscale. Use this configuration with the OTLP exporter and batch processor 
to help with retries for failed writes, and to batch the write requests to Promscale:
```
